airflow-core/src/airflow/serialization/serialized_objects.py

```diff
@@ -1862,10 +1878,39 @@ def _is_excluded(cls, var: Any, attrname: str, op: DAGNode):
         # Check if value matches client_defaults (hierarchical defaults optimization)
         if cls._matches_client_defaults(var, attrname):
             return True
-        schema_defaults = cls.get_schema_defaults("operator")
 
+        # for const fields, we should always be excluded when False, regardless of client_defaults
+        # Use class-level cache for optimisation
+        if not hasattr(cls, "_const_fields"):
+            cls._const_fields = cls.get_operator_const_fields()
```

Comment:
Do we still need the lru cache?

Reply:
Thought I removed it. No we dont need it anymore.

get_operator_const_fields() is only called from one place and its called once per class when initializing cls._const_fields, the class-level cache (cls._const_fields) persists for the class lifetime.

Reply:
Pushed here: [suggestions from ephraim](https://github.com/apache/airflow/pull/57853/commits/ccedbb8808fd2a289e2e0ca30b1dba29b47fdff4)